wandb/sdk/internal/internal_api.py

```diff
@@ -1901,6 +1907,7 @@ def upsert_sweep(
                 entityName: $entityName,
                 projectName: $projectName,
                 controller: $controller,
+                launchScheduler: $launchScheduler,
```

Comment:
I am pretty sure this will break every `wandb server` user who has a build before:
https://github.com/wandb/core/commit/13a1c318816182c25420a6c8f3ef00c2a7e3b508
... which is all of them.

You will want to gate additions to graphql mutations and queries by introspection.

Reply:
Okay fixed it by adding another mutation to the `for mutation in mutations` logic that exists here in the UpsertSweep mutation.